src/solver/variable/storage/minmax-data.cpp

```diff
@@ -25,14 +25,12 @@
 
 namespace Antares::Solver::Variable::R::AllYears
 {
-constexpr double eps = 1.e-7;
-
 static void initArray(bool opInferior, std::vector<MinMaxData::Data>& array)
 {
     for (auto& data: array)
     {
         data.value = opInferior ? DBL_MAX : -DBL_MAX; // +inf or -inf
-        data.indice = (uint32_t)(-1);                 // invalid indice
+        data.indices = 0;
```

Comment:
Revert this line, and `eps`

Reply:
Epsilon is commonly used to check if a value == 0
In this context it created non deterministic results for ids